charts/super-agent/README.md

```diff
@@ -53,74 +61,85 @@ super-agent-deployment:
-| super-agent-deployment.proxy | string | `""` | TODO: Configures the integration to send all HTTP/HTTPS request through the proxy in that URL. The URL should have a standard format like `https://user:password@hostname:port`. Can be configured also with `global.proxy` |
-| super-agent-deployment.rbac.create | bool | `true` | Whether the chart should automatically create the RBAC objects required to run. |
-| super-agent-deployment.resources | object | `{}` | Resource limits to be added to all pods created by the integration. |
-| super-agent-deployment.serviceAccount | object | See `values.yaml` | Settings controlling ServiceAccount creation. |
-| super-agent-deployment.serviceAccount.create | bool | `true` | Whether the chart should automatically create the ServiceAccount objects required to run. |
-| super-agent-deployment.tolerations | list | `[]` | Sets pod's tolerations to node taints. Can be configured also with `global.tolerations` |
-| super-agent-deployment.verboseLog | bool | `false` | Sets the debug logs to this integration or all integrations if it is set globally. Can be configured also with `global.verboseLog` |
+| Key                                                                    | Type   | Default                            | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|------------------------------------------------------------------------|--------|------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Something is weird you have a super long sentence I think

Reply:
The sentence was already there, I just formatted it 🤷‍♂️ 
I'll take a look

Reply:
fixed in d455a3a6c181a915d7c093fdf4d3b3bc0dceddcf